packages/contracts-ops/README.md

```diff
@@ -2,6 +2,31 @@
 
 Contains forge scripts for Nomad system maintenance
 
+## Building Blocks
+
+- `Config.sol` utilities for loading and parsing Nomad Config JSON
+  - initalize the config with the path to a JSON config
+- `CallBatch.sol` utility for aggregating calls into a callbatch, and
+  outputting the batch to JSON
+  - initialize a callbatch with the local domain name and an output path
+  - output paths will be prepended with `./actions/`
+  - call `push(to, data)` to add a call to the batch
+  - call `finish()` to output the list of calls
+  - call `build()` to output the built call to `executeGovernanceActions()`
```

Comment:
note: we'd like two build options - normal governance & recovery

Reply:
yes, that's the behavior described here

Reply:
🤔  no this only builds for recovery. normal governance puts all domains together